client/my-sites/hosting/github/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { useSelector } from 'react-redux';
+import { GithubAuthorizeCard } from 'calypso/my-sites/hosting/github/github-authorize-card/index';
+import { GithubConnectCard } from 'calypso/my-sites/hosting/github/github-connect-card/index';
+import {
+	getKeyringConnections,
+	isKeyringConnectionsFetching,
+} from 'calypso/state/sharing/keyring/selectors';
+import { GitHubPlaceholderCard } from './github-placeholder-card';
+
+export const GitHubCard = () => {
+	const isFetching = useSelector( isKeyringConnectionsFetching );
+	const connections = useSelector( getKeyringConnections );
+
+	if ( ! isEnabled( 'hosting/github-integration' ) ) {
+		return null;
+	}
```

Comment:
Maybe we should also check for the presence of the `github-deploy` service (i.e. that it's returned by the `<QueryKeyringServices>` request). That way we can hide the card using logic on the backend if we want to. We could gate it behind a plan or something like that.